src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -18,6 +18,12 @@ import messages from './messages';
 function LtiConfigForm({
   appConfig, app, onSubmit, intl, formRef, title,
 }) {
+  const ltiAppConfig = {
```

Comment:
emptyAppConfig would make it more clear

Reply:
`emptyAppConfig` will create confusion because first time it would be empty but second time it will populate with saved data.

Reply:
Why this object is needed? 
Is the use case here that the parent component can pass "undefined" values here? if so, can't "default props" help in this case?

Reply:
This object is needed for formik validation. Initially appConfig does not have `consumerKey `, `consumerSecret `, `launchUrl ` and formik does not know about the validation fields. We need to pass field names as initial object so it can know about validation fields.

Reply:
I have to discuss that with you on call.